package.json

```diff
@@ -241,7 +241,7 @@
     "ts-node": "^10.9.1",
     "tsec": "0.2.7",
     "tslib": "^2.6.3",
-    "typescript": "^5.7.0-dev.20240826",
+    "typescript": "^5.7.0-dev.20240827",
```

Comment:
Yeah, it seems like the root package.json version of typescript that is compatible with this python extension update has deviated since our last upstream merge of vscode, so now we need `20240927`

Reply:
I'm nervous about changing the root TS version based on one extension. Could it lead to issues with other built-in extensions? Is it possible to rather pin the Python version to match the root version?

Reply:
I first believed that matching the python extension to root would break the build in the same way as we were seeing in [this CI run](https://github.com/posit-dev/positron/actions/runs/11117084279/job/30888442108?pr=4670), but it seems to build locally. Pushed the change and trying it out now 👀